### PR TITLE
Product startegy and plan refinement

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description

- Updated the Product Strategy towards "Low-Code IIoT Application Platform"
- Introduced Product Plan for Q3 and Q4

### CODE REVIEW

File: src/handbook/product/plan.md [hunk 1]
```diff
@@ -3,9 +3,7 @@ navTitle: Plan
 ---
 # Product Plan
 
-## Background
-
-Our [Product Strategy](./strategy.md) sets out 4 pillars that our product work is focussed around.
+Our [Product Strategy](./strategy.md) sets out 5 pillars that our product work is focussed around.
```
Comment (ZJvandeWeg) [inline src/handbook/product/plan.md:6]: Absolute links > relative links

```suggestion
Our [Product Strategy](/handbook/product/strategy.md) sets out 5 pillars that our product work is focussed around.
```

File: src/handbook/product/plan.md [hunk 2]
```diff
@@ -17,58 +15,53 @@ We use three project boards to plan and track our work.
...
 
-## Principles
+## High-Level Strategy Implementation for Q3/Q4 2023
+
+Our high-level strategy implementation plan serves as a broad outline of the key focus areas for each strategy pillar we will be tackling over the course of the next six months. The points mentioned here are just an overview, and for more granular details on each initiative, we recommend referring to our roadmap.
+
+The progress bars presented adjacent to each category are rough estimates, offering an initial glance at the projected level of completion for each category's planned feature set. These are not precise calculations, nor do they represent the current status of each strategy pillar. It's important to note that at the start of every new product plan cycle, these progress bars reset to a lower level, reflecting the onset of new development and implementation stages. For detailed updates on the actual status, please keep an eye on our [release blog posts](https://flowforge.com/blog/releases/).
+
+### Advancing Enterprise-Readiness 
+![](https://geps.dev/progress/30)
```
Comment (ZJvandeWeg) [inline src/handbook/product/plan.md:25]: Cool stuff

File: src/handbook/product/plan.md [hunk 3]
```diff
@@ -17,58 +15,53 @@ We use three project boards to plan and track our work.
...
+![](https://geps.dev/progress/0)
 
-For every feature we add that has some scope for configuration, our starting point
-in the design is to **identify the right configuration and hard-code it in**.
+- [Official Support for selected Node-RED nodes](https://github.com/flowforge/flowforge/issues/1901)
 
-This removes the choice from users' hands and minds. It does not prevent us from
-choosing to make it more configurable in the future if user-feedback/business-needs
-requires it.
+- [Support for costum NPM Module registry](https://github.com/flowforge/flowforge/issues/217)
```
Comment (ZJvandeWeg) [inline src/handbook/product/plan.md:50]:
```suggestion
- [Support for custom NPM Module registry](https://github.com/flowforge/flowforge/issues/217)
```

File: src/handbook/product/plan.md [hunk 4]
```diff
@@ -17,58 +15,53 @@ We use three project boards to plan and track our work.
...
-to ensure a consistent user experience across FlowForge instances.
+### Data Visualization & Analytics
+![](https://geps.dev/progress/5)
 
-Options that have an impact on the behaviour of flows will need to be considered
-on a case-by-case basis. But the starting point should always be to pick the right
-default and only expose to the user if absolutely necessary.
+- Version 1.0 of the Node-RED Dashboard successor, with a comparable feature set like [Node-RED Dashboard](https://github.com/node-red/node-red-dashboard).
 
+- FlowForge Dashboard extesnion (Version 1.0) allowing [user specific dashboards](https://github.com/flowforge/flowforge-nr-dashboard/issues/1) and RBAC.
```
Comment (ZJvandeWeg) [inline src/handbook/product/plan.md:60]:
```suggestion
- FlowForge Dashboard extension (Version 1.0) allowing [user specific dashboards](https://github.com/flowforge/flowforge-nr-dashboard/issues/1) and RBAC.
```

File: src/handbook/product/strategy.md [hunk 5]
```diff
@@ -4,59 +4,44 @@ navTitle: Strategy
 
 # Product Strategy
 
-Our strategy is an indication and statement of intent for the next 2 years, and reflects FlowForge’s vision. It’s a draft, as everything in the handbook is a draft, so we will revisit regularly and iterate when there is a reason to do so. FlowForge is a 100 percent agile company, so our product is; timing is everything. Have a look at our roadmap for what we're pursuing in the upcoming releases. 
+Our strategy is an indication and statement of intent for the next 2 years, and reflects FlowForge’s vision, to become the leading "Low-Code IIoT Application Platform".
+It’s a draft, as everything in the handbook is a draft, so we will revisit regularly and iterate when there is a reason to do so. FlowForge is a 100 percent agile company, so our product is; timing is everything.
+Have a look at our [roadmap](https://flowforge.com/product/roadmap/) for what we're pursuing in the upcoming releases.
```
Comment (ZJvandeWeg) [inline src/handbook/product/strategy.md:9]:
```suggestion
Have a look at our [roadmap](/product/roadmap/) for what we're pursuing in the upcoming releases.
```

File: src/handbook/product/strategy.md [hunk 6]
```diff
@@ -4,59 +4,44 @@ navTitle: Strategy
...
 
-### FlowForge for enterprise
+### Advancing Enterprise-Readiness
 
-The cornerstone of FlowForge is to make Node-RED more accessible to enterprises. Therefore, we are constantly implementing requirements that support the easy use of Node-RED in production environments. We will focus on:
+FlowForge is committed to augmenting the enterprise-readiness of Node-RED. Our strategy emphasizes enhancing audit and compliance capabilities, instituting stringent preventive and corrective controls, and bolstering security. By integrating these vital aspects, we aim to create a reliable, secure, and scalable environment. This initiative underlines our dedication to surpassing enterprise expectations and strengthening Node-RED's position in the industry.
```
Comment (ZJvandeWeg) [inline src/handbook/product/strategy.md:13]:
```suggestion
FlowForge is committed to augmenting the enterprise-readiness of Node-RED. Our strategy emphasises enhancing audit and compliance capabilities, instituting stringent preventive and corrective controls, and bolstering security. By integrating these vital aspects, we aim to create a reliable, secure, and scalable environment. This initiative underlines our dedication to surpassing enterprise expectations and strengthening Node-RED's position in the industry.
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

**Comment by ZJvandeWeg** [comment]
@MarianRaphael Had a few notes now, but want to take some time to think about this before merging.

File: src/handbook/product/strategy.md (hunk 6 above)
Comment (MarianRaphael) [reply src/handbook/product/strategy.md:13]: According to https://flowforge.com/handbook/marketing/website/#website
"All written content should be in UK English." For me, both options are fine, I just need to know.

**Review by ZJvandeWeg — DISMISSED** [review]

**Comment by ZJvandeWeg** [comment]
@MarianRaphael This is still a draft, I think it should be marked as ready to review?

**Review by knolleary — APPROVED** [review]

**Review by iskerrett — APPROVED** [review]

**Pull request merged**